client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -234,6 +236,15 @@ export default function getThankYouPageUrl( {
 		);
 	}
 
+	// Asismet site-less checkout
+	if ( isAkismetSitelessCheckout ) {
+		// extract a product from the cart, in siteless checkout there should only be one
+		const productSlug = cart?.products[ 0 ]?.product_slug ?? 'no_product';
+
+		debug( 'redirecting to siteless Akismet thank you' );
+		return `/checkout/akismet/thank-you/${ productSlug }`;
```

Comment:
I believe we have a task set for this here: 1204136657007202-as-1204136663449775/f

Since this PR will be feature flagged when deployed, I think it makes sense to do this separately

Correct me if I am wrong @jboland88

Reply:
Aha - yes, there is a separate task to address the thank you page.
I had added the route handling here initially so that the checkout would go to a known place where the Akismet thank you page will be in the future. 
Not opposed to scratching the thank you page in this PR and handling that in the separate task. OR if we can get the unit tests satisfied for the blank route - either seems OK IMO. WDYT @CodeyGuyDylan ?

Reply:
It couldn't hurt to add the tests here, since it should be too much work to do so. I think i'll go ahead and do that